core/src/xframe/core/alpha.cljc

```diff
@@ -124,7 +124,7 @@
                ~get-state-sym #(<- ~s-sym k#)]
            ~(if (uix.lib/cljs-env? &env)
               `(if ~(with-meta 'goog.DEBUG {:tag 'boolean})
-                 (if (and ~'js/__REACT_DEVTOOLS_GLOBAL_HOOK__
+                 (if (and (some? js/window.__REACT_DEVTOOLS_GLOBAL_HOOK__)
```

Comment:
This will fail when running on Node.js or React Native
```suggestion
                 (if (and (some? (.-__REACT_DEVTOOLS_GLOBAL_HOOK__ goog.global))
```